### PR TITLE
Added default keybindings for panes

### LINKED ISSUE #3544: Set panes keybindings by default
# Description of the new feature/enhancement
Include panes keybindings by default

# Proposed technical implementation details (optional)
Current keybinding suggestion:
```       
		{ "command": "splitHorizontal", "keys": [ "alt+shift+-" ] },
        { "command": "splitVertical", "keys": [ "alt+shift+plus" ] },
        { "command": { "action": "moveFocus", "direction":"down" }, "keys": [ "alt+down" ] },
        { "command": { "action": "moveFocus", "direction":"left" }, "keys": [ "alt+left" ] },
        { "command": { "action": "moveFocus", "direction":"right" }, "keys": [ "alt+right" ] },
        { "command": { "action": "moveFocus", "direction":"up" }, "keys": [ "alt+up" ] },
        { "command": { "action": "resizePane", "direction":"down" }, "keys": [ "alt+shift+down" ] },
        { "command": { "action": "resizePane", "direction":"left" }, "keys": [ "alt+shift+left" ] },
        { "command": { "action": "resizePane", "direction":"right" }, "keys": [ "alt+shift+right" ] },
        { "command": { "action": "resizePane", "direction":"up" }, "keys": [ "alt+shift+up" ] },
```

### PR DESCRIPTION
**Pull request by cinnamon-msft**
## Summary of the Pull Request
Added the following keybindings for panes:
```       
	{ "command": "splitHorizontal", "keys": [ "alt+shift+-" ] },
        { "command": "splitVertical", "keys": [ "alt+shift+plus" ] },
        { "command": { "action": "moveFocus", "direction":"down" }, "keys": [ "alt+down" ] },
        { "command": { "action": "moveFocus", "direction":"left" }, "keys": [ "alt+left" ] },
        { "command": { "action": "moveFocus", "direction":"right" }, "keys": [ "alt+right" ] },
        { "command": { "action": "moveFocus", "direction":"up" }, "keys": [ "alt+up" ] },
        { "command": { "action": "resizePane", "direction":"down" }, "keys": [ "alt+shift+down" ] },
        { "command": { "action": "resizePane", "direction":"left" }, "keys": [ "alt+shift+left" ] },
        { "command": { "action": "resizePane", "direction":"right" }, "keys": [ "alt+shift+right" ] },
        { "command": { "action": "resizePane", "direction":"up" }, "keys": [ "alt+shift+up" ] },
```

 
## PR Checklist
* [x] Closes #3544
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [x] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

### CODE REVIEW

File: src/cascadia/TerminalApp/defaults.json [hunk 1]
```diff
@@ -200,41 +200,51 @@
...
+      { "command": "newTab", "keys": [ "ctrl+shift+t" ] },
+      { "command": "newTabProfile0", "keys": [ "ctrl+shift+1" ] },
+      { "command": "newTabProfile1", "keys": [ "ctrl+shift+2" ] },
+      { "command": "newTabProfile2", "keys": [ "ctrl+shift+3" ] },
+      { "command": "newTabProfile3", "keys": [ "ctrl+shift+4" ] },
+      { "command": "newTabProfile4", "keys": [ "ctrl+shift+5" ] },
+      { "command": "newTabProfile5", "keys": [ "ctrl+shift+6" ] },
+      { "command": "newTabProfile6", "keys": [ "ctrl+shift+7" ] },
+      { "command": "newTabProfile7", "keys": [ "ctrl+shift+8" ] },
+      { "command": "newTabProfile8", "keys": [ "ctrl+shift+9" ] },
```
Comment (DHowett-MSFT) [inline src/cascadia/TerminalApp/defaults.json]: why does this revert these to the legacy key bindings without args..?

Comment (cinnamon-msft) [reply src/cascadia/TerminalApp/defaults.json]: Oh that's my bad, didn't notice the change in master

**Comment by bchap1n** [comment]
is there a way to close a split pane? It seems too easy to open one too many panes.

**Comment by zadjii-msft** [comment]
@bchap1n The default keybinding for "Close Tab" (<kbd>ctrl+shift+w</kbd>) is actually secretly bound to "Close Pane". It just so happens that when you have only one pane in a tab, it'll close the pane too 😜

**Comment by mcornella** [comment]
@DHowett-MSFT the default keybinding for "split vertically" says to use `=` (equal) instead of `+` (plus) in the [release changelog](https://github.com/microsoft/terminal/releases/tag/v0.7.3291.0).

**Comment by DHowett-MSFT** [comment]
Yes, I erroneously oversimplified it based on my US 104-key keyboard (where it is impossible to press <kbd>Alt+Shift+Plus</kbd> without pressing <kbd>Alt+Shift+=</kbd>)

**Pull request merged**